Formula/xmrig.rb

```diff
@@ -11,16 +11,23 @@ class Xmrig < Formula
     sha256 "3a274f5c98cd7fb6946e208bf2b4e1a5309ce43f92e937208e188324bef3a937" => :el_capitan
   end
 
+  option "without-httpd", "Build without built in http server and API"
+  option "without-aeon", "Disable CryptoNight-Lite support"
+
   depends_on "cmake" => :build
-  depends_on "libmicrohttpd"
+  depends_on "libmicrohttpd" if build.with? "httpd"
```

Comment:
`if build.with?` deps are grim outside of `def install`. If you absolutely need it the option should be called `"without-libmicrohttpd"`.

Reply:
I have not much concern on naming that option to be longer or shorter, but that name is sort of same as the cmake option.